api/v1/submissions/PKPSubmissionHandler.inc.php

```diff
@@ -171,6 +202,11 @@ public function __construct()
                     'handler' => [$this, 'deletePublication'],
                     'roles' => [Role::ROLE_ID_MANAGER, Role::ROLE_ID_SUB_EDITOR, Role::ROLE_ID_ASSISTANT],
                 ],
+                [
+                    'pattern' => $this->getEndpointPattern() . '/{submissionId:\d+}/publications/{publicationId:\d+}/contributors/{contributorId:\d+}',
+                    'handler' => [$this, 'deleteContributor'],
+                    'roles' => [Role::ROLE_ID_MANAGER, Role::ROLE_ID_SUB_EDITOR, Role::ROLE_ID_ASSISTANT],
+                ],
```

Comment:
This is great. We'll need to expand our `swagger-source.json` file to document each of these API endpoints. Maybe we can work on this during a pair programming session.